float32  dtheta_rad
float32  ang_speed
---
bool success
